arbnode/batch_poster.go

```diff
@@ -1826,16 +1830,7 @@ func (b *BatchPoster) Start(ctxIn context.Context) {
 		espressoEphemeralErrorHandler.Reset()
 	}
 	b.CallIteratively(func(ctx context.Context) time.Duration {
-		var err error
-		espresso, _ := b.streamer.isEspressoMode()
-		if !espresso {
-			if b.streamer.lightClientReader != nil && b.streamer.espressoClient != nil {
```

Comment:
why are we deleting this check?

Reply:
Well this check only happens when the batch poster is started to prevent it from posting any data until the chain config was updated. Given we don't populate this at runtime anymore, the check doesn't do anything meaningful anymore.